tools/slang-test/slang-test-main.cpp

```diff
@@ -1132,7 +1132,7 @@ static SlangResult _extractRenderTestRequirements(const CommandLine& cmdLine, Te
             break;
         case RenderApiType::WebGPU:
             target = SLANG_WGSL;
-            SLANG_ASSERT(!usePassthru);
+            //SLANG_ASSERT(!usePassthru);
```

Comment:
Please don't just disable this, if possible. I have a bug for investigating this:
https://github.com/shader-slang/slang/issues/5276

Last I checked this only triggers in a single test, so that test should then be disabled.

Reply:
I think we can just set passThru to `SLANG_PASS_THROUGH_NONE` here.

Reply:
I think I now have a proper fix for this.
But it will be on another PR.
Marking this as resolved.